tools/tsp-client/src/index.ts

```diff
@@ -182,6 +184,28 @@ async function generate({
   }
 }
 
+
+async function convert(readme: string, outputDir: string): Promise<void> {
+  return new Promise((resolve, reject) => {
+      const autorest = spawn("autorest", [readme, "--openapi-to-typespec", "--use=@autorest/openapi-to-typespec", `--output-folder=${outputDir}`], {
```

Comment:
Should you maybe check if autorest is  installed? or maybe can use `npx autorest` to download on the fly

Reply:
I was going to put autorest as a prerequisite for install on the instructions...but I think npx autorest would be nice too, I'll switch it up to npx autorest